### PR TITLE
PYIC-2419 Add audit event delete

### PR DESCRIPTION
**Pull request by blakeyp**
### What changed

Send audit event `IPV_DELETE_USER_DATA` when lambda executes successfully and has deleted some data. Include `source-topic-arn` which identifies the SNS topic the message came from.

### Why did it change

So we have audit logs for deleted data

### Issue tracking
- [PYIC-2419](https://govukverify.atlassian.net/browse/PYIC-2419)

[PYIC-2419]: https://govukverify.atlassian.net/browse/PYIC-2419?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lambdas/delete-user-data/src/config.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
 export const config = {
+  componentId: "ipv-core-delete-user-data",
```
Comment (kerrr) [inline lambdas/delete-user-data/src/config.ts]: `componentId` should be the URI representing core. In production this is https://identity.account.gov.uk and is the config param at `core/self/componentId`

**Comment by kerrr** [comment]
Have you contacted TxMA about adding a new audit event?

**Comment by blakeyp** [comment]
> Have you contacted TxMA about adding a new audit event?

Haven't yet, will do

File: lambdas/delete-user-data/src/config.ts (hunk 1 above)
Comment (blakeyp) [reply lambdas/delete-user-data/src/config.ts]: Changed this to now get the componentId from SSM

**Review by kerrr — APPROVED** [review]

**Pull request merged**